### ISSUE TITLE
Support install docker by rancher/install-docker on opensuse/sels

### ISSUE THREAD

**Issue by futuretea** [rancher/rancher#31817]
**What kind of request is this (question/bug/enhancement/feature request):**
feature request

**Steps to reproduce (least amount of steps as possible):**
1. create a opensuse/sles vm
2. run `curl -sL https://releases.rancher.com/install-docker/20.10.sh | sh -`

**Result:**
if use opensuse image ( openSUSE-Leap-15.2-JeOS.x86_64-15.2-OpenStack-Cloud-Build31.234.qcow2)
```bash
opensuse@vm1:~> cat /etc/os-release
NAME="openSUSE Leap"
VERSION="15.2"
ID="opensuse-leap"
ID_LIKE="suse opensuse"
VERSION_ID="15.2"
PRETTY_NAME="openSUSE Leap 15.2"
ANSI_COLOR="0;32"
CPE_NAME="cpe:/o:opensuse:leap:15.2"
BUG_REPORT_URL="https://bugs.opensuse.org"
HOME_URL="https://www.opensuse.org/"
```
```bash
opensuse@vm1:~> curl -sL https://releases.rancher.com/install-docker/20.10.sh | sh -
# Executing docker install script, commit: 3d8fe77c2c46c5b7571f94b42793905e5b3e42e4

ERROR: Unsupported distribution 'opensuse-leap'
```

if use sels image (SLES15-SP2-JeOS.x86_64-15.2-OpenStack-Cloud-GM.qcow2)
```bash
suse@vm2:~> cat /etc/os-release
NAME="SLES"
VERSION="15-SP2"
VERSION_ID="15.2"
PRETTY_NAME="SUSE Linux Enterprise Server 15 SP2"
ID="sles"
ID_LIKE="suse"
ANSI_COLOR="0;32"
CPE_NAME="cpe:/o:suse:sles:15:sp2"
```
```bash
suse@vm2:~> curl -sL https://releases.rancher.com/install-docker/20.10.sh | sh -
# Executing docker install script, commit: 3d8fe77c2c46c5b7571f94b42793905e5b3e42e4

ERROR: Unsupported distribution 'sles'
```

**Comment by superseb** [comment]
This was fixed in https://github.com/rancher/rancher/issues/25071 for 19.03.x as SUSE packages were only available for 19.03.x. Are there SUSE packages provided for 20.10.x?

**Comment by futuretea** [comment]
> This was fixed in #25071 for 19.03.x as SUSE packages were only available for 19.03.x. Are there SUSE packages provided for 20.10.x?

opensuse ( openSUSE-Leap-15.2-JeOS.x86_64-15.2-OpenStack-Cloud-Build31.234.qcow2)
```bash
suse@vm1:~> curl -sL https://releases.rancher.com/install-docker/19.03.sh | sh -

Either your platform is not easily detectable or is not supported by this
installer script.
Please visit the following URL for more detailed installation instructions:

https://docs.docker.com/engine/installation/
```
sles (SLES15-SP2-JeOS.x86_64-15.2-OpenStack-Cloud-GM.qcow2)
```bash
suse@vm1:~> curl -sL https://releases.rancher.com/install-docker/19.03.sh | sh -

Either your platform is not easily detectable or is not supported by this
installer script.
Please visit the following URL for more detailed installation instructions:

https://docs.docker.com/engine/installation/
```

**Comment by superseb** [comment]
https://github.com/rancher/rancher/issues/25071#issuecomment-630429914

`19.03.5` was the version added for SLES
